graphql-language/src/ast.rs

```diff
@@ -59,20 +60,18 @@ impl_node_for! { FloatValue }
 ///   - `""`
 ///   - `"` StringCharacter+ `"`
 pub struct StringValue<'a> {
-  pub loc: Option<Location<'a>>,
-  pub value: String
+  pub loc: Location<'a>,
+  pub value: &'a str
```

Comment:
The String token only capture the raw string from source, the parser required to unescape the string (e.g. \' -> ') and validate and convert Unicode escape chars (e.g. \u2764 ->  '❤'). Therefore, the parser output of StringValue will not reference to the source.